style.css

```diff
@@ -0,0 +1,207 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+}
+
+.text-color {
+  color: white;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 10%;
+  margin-left: 10%;
+  align-items: center;
+  padding: 6px;
+}
+
+.main-header {
+  background-color: #3c3a39;
+  width: 100vw;
+}
+
+.menubar {
+  height: 20px;
+}
+
+.section-1 {
+  background-color: #1c1a19;
+  height: 100vh;
+  width: 100vw;
+}
+
+.content-0 {
+  width: 100vw;
+  height: 100vh;
+  padding-top: 37%;
+  background-image: url(./socialmedia/design.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+}
+
+.heading-1-container {
+  margin-bottom: 20px;
+}
+
+.margin-24 {
+  margin: 0 24px;
+}
+
+.heading-1 {
+  color: white;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal, s;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.heading-2 {
+  margin-left: 24px;
+  color: white;
```

Comment:
Please, use the good color to stick Figma design.
The good color here is #FFF5E1